client/reader/following-edit/style.scss

```diff
@@ -44,6 +44,10 @@
 	float: right;
 }
 
+.is-reader-page .search-card.card {
+	box-shadow: 0 0 0 2px rgba(200, 215, 225, 0.5), 0 1px 2px #e9eff3;
```

Comment:
thanks guys! I took the colors from the Chrome inspector without realizing it was compiled from variables!

fixing!